src/include/network/routines.rb

```diff
@@ -32,6 +32,58 @@ module NetworkRoutinesInclude
     include Yast
     include Logger
 
+    CONTROLLERS = {
+      2   => "network",
+      7   => "communication",
+      12  => "infiniband",
+      258 => "modem",
+      259 => "isdn",
+      276 => "dsl",
+      263 => "interface"
+    }
+
+    SUBCLASSES = {
+      "communication" => {
+        3 => "modem"
+      },
+      "network"       => {
+        0   => "eth",
+        1   => "tr",
+        2   => "fddi",
+        3   => "atm",
+        4   => "isdn",
+        6   => "ib",
+        7   => "ib",
+        129 => "myri",
+        130 => "wlan",
+        131 => "xp",
+        134 => "qeth",
+        135 => "hsi",
+        136 => "ctc",
+        137 => "lcs",
+        142 => "ficon",
+        143 => "escon",
+        144 => "iucv",
+        145 => "usb" # #22739
+      },
+      "interface"     => {
+        0   => "lo",
+        1   => "eth",
+        2   => "tr",
+        3   => "fddi",
+        4   => "ctc",
+        5   => "iucv",
+        6   => "hsi",
+        7   => "qeth",
+        8   => "escon",
+        9   => "myri",
+        10  => "wlan",
+        11  => "xp",
+        12  => "usb",
+        129 => "sit"
+      }
+    }
```

Comment:
I'm thinking if these constants belongs to a yaml file. These constants are defined by hw standards and putting it into the code makes the file unnecessarily long.